frontend/public/components/monitoring/alert-manager-utils.tsx

```diff
@@ -4,28 +4,39 @@ import { Base64 } from 'js-base64';
 import { safeLoad, safeDump } from 'js-yaml';
 
 import { k8sPatch, K8sResourceKind } from '../../module/k8s';
-import {
-  AlertManagerConfig,
-  AlertManagerReceiver,
-  AlertManagerRoute,
-} from './alert-manager-config';
+import { AlertManagerConfig } from './alert-manager-config';
 import { SecretModel } from '../../models';
 
-export const DEFAULT_RECEIVER_LABEL = 'All (default receiver)';
+export const receiverTypes = {
+  pagerduty_configs: 'PagerDuty',
+  webhook_configs: 'Webhook Receiver',
+};
 
-export const getAlertManagerConfig = (secret: K8sResourceKind, setErrorMsg) => {
+export const getAlertManagerYAML = (secret: K8sResourceKind, setErrorMsg) => {
   const alertManagerYaml = _.get(secret, ['data', 'alertmanager.yaml']);
-  let config: AlertManagerConfig;
+  let yaml: string = '';
```

Comment:
nit: the type is inferred, so `: string` isn't needed

```suggestion
  let yaml = '';
```

Reply:
fixed